neuro_flow/expr.py

```diff
@@ -701,14 +725,66 @@ def a(value: str) -> Parser:
 
 FUNC_CALL: Final = (NAME + LPAR + FUNC_ARGS + RPAR + TRAILER) >> make_call
 
+ATOM_EXPR.define(LPAR + EXPR + RPAR | ATOM | FUNC_CALL | LOOKUP)
+
+# Here we define operator precedence:
+# +, - (unary)
+# *, /
+# +, -
+# |
+# ==, !=, <, >, <=, =>
+# and
+# or
+
+# For operators, LL1 grammar rules are used to speed-up parsing
+
+DISJUNCTION: Final = forward_decl()
+DISJUNCTION_TRAILER: Final = forward_decl()
+CONJUNCTION: Final = forward_decl()
+CONJUNCTION_TRAILER: Final = forward_decl()
+INVERSION: Final = forward_decl()
+COMPARISON: Final = forward_decl()
+COMPARISON_TRAILER: Final = forward_decl()
+BITWISE_OR: Final = forward_decl()
```

Comment:
I can imagine a naming problem in the future if we add _true bitwise ops_ like `&` and `~`.
Not a big deal though, all names are private.

Reply:
I used naming from here: https://docs.python.org/3/reference/grammar.html